src/impl.rs

```diff
@@ -161,3 +161,32 @@ impl<T: ?Sized + Any> Miny<T> {
 		}
 	}
 }
+
+#[cfg(feature = "allocative")]
+impl<T> allocative::Allocative for Miny<T>
+where
+	T: ?Sized + allocative::Allocative,
+{
+	fn visit<'a, 'b: 'a>(&self, visitor: &'a mut allocative::Visitor<'b>) {
```

Comment:
more descriptive names for the lifetimes? although these are the names the trait definition uses…